service/src/test/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiControllerGcsBucketTest.java

```diff
@@ -183,10 +183,6 @@ public void setup() throws Exception {
         userAccessUtils.defaultUser().getGoogleCredentials(), projectId);
     GcpCloudUtils.waitForProjectAccess(
         userAccessUtils.defaultUser().getGoogleCredentials(), projectId2);
-    GcpCloudUtils.waitForProjectAccess(
```

Comment:
we need both these waits for environments where we can't give temporary grants to test users (like Verily environments). Otherwise, operations that need end-user credentials on the cloud will flake

Reply:
I think that was a merge mistake. I didn't intend to change that test!